deploy/helm/sumologic/values.yaml

```diff
@@ -210,13 +203,20 @@ fluent-bit:
     systemd:
       enabled: true
 
+  parsers:
+    enabled: true
+    regex:
+      - name: multi_line
+        regex: (?<log>.*\d{4}-\d{1,2}-\d{1,2} \d{2}:\d{2}:\d{2}.*)
```

Comment:
can we add a comment above this with the format it will match, something like
```
# Matches "2019-11-17 07:14:12"
```

Reply:
Also how does this handle a timestamp with that format appearing somewhere in the middle of a multi-line message? Will it split it incorrectly?

Reply:
We can change the regex to ```(?<log>^{"log":"\d{4}-\d{1,2}-\d{1,2} \d{2}:\d{2}:\d{2}.*)``` to avoid that case.